gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManagerUtils.java

```diff
@@ -219,6 +219,7 @@ public static Set<DagNode<JobExecutionPlan>> getNext(Dag<JobExecutionPlan> dag)
         switch (failureOption) {
           case FINISH_RUNNING:
             return new HashSet<>();
+          // todo - FINISH_ALL_POSSIBLE should probably `continue` not `break`
```

Comment:
seems a simple enough change... modify now?

Reply:
this is a legacy code and very important. I would like to not touch it in this PR because this PR is addressing an alarming issue and cannot afford introducing issue.
It is also not related to this PR work, so will fix it in a separate PR